storm-core/src/clj/backtype/storm/log.clj

```diff
@@ -44,3 +46,11 @@
 (defn log-stream
   [& args]
   (apply log/log-stream args))
+
+(defmacro pprint-message
```

Comment:
I would prefer to have the name log somewhere in the function name.  perhaps log-pprint